app/models/restricted_image.rb

```diff
@@ -0,0 +1,22 @@
+# The type of image that is used if a user doesn't have
+# `can? :download, stacks_image' permissions
+class RestrictedImage < StacksImage
```

Comment:
I wonder if it'd help to distinguish tile-only vs thumbnail-only images? I know we weren't necessarily doing that before, but I had trouble grokking what a `RestrictedImage` was at first.

Reply:
To be honest, I'm not really sure what it is about either... this was just a way to keep the models from being aware of `Ability`.  It was clear that we had two classes here with lots of `if`s making it clear there was want of another class.